prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -80,17 +80,21 @@ class FHIRConverter(
                 )
 
                 // create item lineage
+                bundleIndex++
+
                 report.itemLineages = listOf(
                     ItemLineage(
                         null,
                         message.reportId,
-                        bundleIndex++,
+                        bundleIndex,
                         report.id,
                         1,
                         null,
                         null,
                         null,
-                        report.getItemHashForRow(1)
+                        report.getItemHashForRow(1),
+                        message.reportId,
```

Comment:
this is my own ignorance, but is it true that this will always be the first item in the lineage?  In other words, there is not a previous report?

Reply:
This makes sense to me, but is there somewhere we could add a test to verify this functionality?

Reply:
Since this is UP, I'm reverting it to just be null